frontend/src/components/Register/Register.css

```diff
@@ -1,13 +1,89 @@
-.register-form-container{
-    margin-top: 20px;
+@import url('http://fonts.cdnfonts.com/css/abeezee');
+
+*{
+    font-family: 'ABeeZee', sans-serif;
 }
-.register-form{
-    width: 30vw;
-    margin: auto;
-    padding: 20px;
+
+.container{
```

Comment:
Let's be more specific with the name to avoid conflicts on a future. Container seems to general and can lead to confusion and unwanted changes on other components when the app grows.